### PR TITLE
Remove src/main/webapp/dev/ folder #4802

### LINKED ISSUE #4802: Remove src/main/webapp/dev/ folder
Previously used to contain the developer documentations. Since they've been migrated (well, most of them) either to Google docs or GitHub markdown, this folder is not necessary anymore. The contents inside, if still used, can be moved to the relevant folders.

Suggestions:
- `mashup.jsp` to the root `src/main/webapp/` folder, `mashup.css` to `src/main/webapp/stylesheets`.
- The contents of `src/main/webapp/dev/images` to `devdocs/images` (make a new folder).
- The link to the JSP style guide: https://docs.google.com/document/d/14bXfdveXvoIaPBYpL19m4PK6oPabSnnoawj6OGjOzD4/pub?embedded=true
- Everything else can be removed.

### PR DESCRIPTION
**Pull request by mikelimantara**
Fixes #4802

### CODE REVIEW

File: devdocs/design.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ##Architecture
 
-![High Level Architecture](https://raw.githubusercontent.com/TEAMMATES/repo/master/src/main/webapp/dev/images/highlevelArchitecture.png)
+![High Level Architecture](https://raw.githubusercontent.com/TEAMMATES/repo/master/devdocs/images/highlevelArchitecture.png)
```
Comment (wkurniawan07) [inline devdocs/design.md]: Relative paths work on `process.md`, it should work here as well. In particular, the images should be correctly referenced in [this file](https://github.com/mikelimantara/repo/blob/4802-remove-src-main-webapp-dev-folder/devdocs/design.md).

**Comment by wkurniawan07** [comment]
One minor improvement requested.

**Comment by mikelimantara** [comment]
Updated the links in design.md to use relative paths as requested. Ready for review.

**Comment by wkurniawan07** [comment]
Good work :+1:

**Pull request merged**